app/templates/play/level/control-bar-view.pug

```diff
@@ -45,9 +45,12 @@ if includeHomeLink
   if spectateGame
     button.btn.btn-xs.btn-inverse.banner#next-game-button(data-i18n="play_level.next_game") Next game
 
-  if !observing
-    button.btn.btn-xs.btn-primary.banner#level-done-button(data-i18n="play_level.done") Done
-
+  //- if !observing
```

Comment:
what is this and why are we removing it? I couldn't find the Done button but I'm not sure if we want to remove it from certain levels that have them

Reply:
I found this line at PlayLevelView.js line 966 (at the app folder one)
```
    $('#level-done-button').show() unless @level.isType('hero', 'hero-ladder', 'hero-coop', 'course', 'course-ladder', 'game-dev', 'web-dev', 'ladder')  # TODO: do we ever use this? Should remove if not.
```
are there any level types beside those ones?

Reply:
Hmm so this will show in these level types
['campaign',  'intro']

Reply:
Alright I put back the old button and conditionally syled it based on if its junior or not, thanks Saurabh! feel free to review it further